### ISSUE TITLE
Open Bletchley?

### ISSUE THREAD

**Issue by lnaie** [Azure/azure-blockchain-projects#52]
Any plans to share the code or provide the whole platform to test/use on premises?

Thanks

**Comment by marleyg** [comment]
Yes.  We are in an early private preview of the Cryptlet Framework and will look to open it up more broadly after this latest crank of the EAP.  After the summer most likely.
